integration_test/Bindings/Python/dialects/msft.py

```diff
@@ -85,13 +85,6 @@
   assert num_failed == 1
   # ERR: error: 'hw.instance' op Could not apply placement #msft.physloc<M20K, 2, 6, 1>. Position already occupied by hw.instance "ext1" @MyExternMod
 
-  # CHECK-LABEL: === tcl ===
-  print("=== tcl ===")
-
-  # CHECK: proc top_config { parent } {
-  # CHECK:   set_location_assignment M20K_X2_Y6_N1 -to $parent|inst1|ext1|ext1|foo_subpath
```

Comment:
Can you keep these filecheck lines?

Reply:
These depend on the now-removed `msft.export_tcl` API. I can run the lower-msft-to-tcl pass and check for the output that way.

Reply:
Please do

Reply:
So now that I'm looking at this, this test is building a mix of HW and MSFT dialect ops, which isn't compatible with the contract lower-msft-to-tcl expects. I can re-work this test to use only MSFT ops in order to test the Tcl output, but that is a bigger change. Or I can just leave off the Tcl output checks, as it is now.

Reply:
Ok, I've untangled the issue and can add back these checks, they will just have to go at the end of the file, since we modify the IR with lower-msft-to-hw before we can export tcl.